nginx-photon/v2.10.2/rockcraft.yaml

```diff
@@ -0,0 +1,49 @@
+name: nginx
```

Comment:
As mentioned, I'd suggest we stick to the `Version > Component > rockcraft.yaml` folder structure, instead of `Component > Version > rockcraft.yaml` [as seen in the Loki ROCK](https://github.com/canonical/loki-rock/).

Reply:
Good point!